docs/user-guide/src/reference/bmo.md

```diff
@@ -0,0 +1,4 @@
+# Bare Metal Operator Reference
```

Comment:
Why create new pages for 2 lines of information? Let's maybe consolidate all of these under the parent "References" document?

Reply:
I assumed there would be more in the future but if that's not the case are you asking that only a Reference be created and all 3 resources be under that one page?